api/filter/http_fault.proto

```diff
@@ -0,0 +1,93 @@
+syntax = "proto3";
+
+package envoy.api.v2.filter;
+
+import "api/rds.proto";
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/struct.proto";
+import "google/protobuf/wrappers.proto";
+
+// Delay specification is used to inject latency into the rpc operations.
+message Delay {
```

Comment:
Most of this is implemented, except for the exponential faults.. It was there in Istio spec. Just thought I would add it here as well. LMK if you want me to remove it.

Reply:
Either remove or add a tracking issue at https://github.com/envoyproxy/envoy for implementation side (and reference here).